src/app/space/analyze/analyze-overview/analyze-overview.component.html

```diff
@@ -2,24 +2,28 @@
   <div id="analyze-overview-dashboard" class="container-fluid analyze-overview-wrapper" user-level>
     <div class="row f8-dashboard-masthead">
       <div class="col-sm-10">
-        <fabric8-edit-space-description-widget></fabric8-edit-space-description-widget>
+        <fabric8-edit-space-description-widget [userOwnsSpace]="userOwnsSpace()"></fabric8-edit-space-description-widget>
       </div>
       <div class="col-sm-2">
         <f8-feature-toggle featureName="AppLauncher">
-          <button *ngIf="this.authentication.isLoggedIn()" id="analyze-overview-dashboard-add-to-space-button" class="btn btn-primary btn-lg f8-dashboard-masthead-create-application pull-right" (click)="openForgeWizard(addSpace)" default-level> Create an Application </button>
-          <button *ngIf="this.authentication.isLoggedIn()" class="btn btn-primary btn-lg f8-dashboard-masthead-create-application pull-right" (click)="showAddAppOverlay()" user-level> Create an Application </button>
+          <div *ngIf="userOwnsSpace()">
+            <button *ngIf="this.authentication.isLoggedIn()" id="default-level-analyze-overview-dashboard-create-space-button" class="btn btn-primary btn-lg f8-dashboard-masthead-create-application pull-right"
+              (click)="openForgeWizard(addSpace)" default-level> Create an Application </button>
+            <button *ngIf="this.authentication.isLoggedIn()" id="user-level-analyze-overview-dashboard-create-space-button" class="btn btn-primary btn-lg f8-dashboard-masthead-create-application pull-right"
```

Comment:
We should avoid assigning IDs. Especially when only used for testing purposes. Consider using a custom attribute such as `data-tid` (tid = test id).  You can write a selector such as `*[data-tid=foo]`. We could take this one step further and probably even strip these attributes in the future from production builds.

Reply:
Would you mind elaborating a bit more on why its bad to test that way ?

Reply:
There's nothing wrong with doing it this way and I won't disapprove the change as it stands now. When using IDs we must make them globally unique (Which you have done). But for a test id in a custom attribute we only need them unique within the component. We should establish a pattern to follow when creating test IDs for unit or automated tests.
I wasn't sure if it had ever been discussed for osio or not.

I'll bring it up separately as another tech debt item with the team.

Reply:
ID's should only be used for test references, at least as much as possible. There are a few instances where you need it for something like a form label. A Test ID is fine for tests too, though we haven't done it as we don't use ID's for references in the code.